modules/controls.js

```diff
@@ -36,35 +36,43 @@ displaySystem.registerModule({
             system.ws.sendMessage({name:name},action,data);
         }
 
-        function renderModuleControls(name, document) {
-            var module = system.modules[name];
-            if (module.hidden) {
-                return;
+        function handleButton(module, f, inputs) {
+            return () => {
```

Comment:
This is relatively modern stuff (fat-arrow).
I'm fine with it, but it may break support for older browsers.

Since we don't mention anything about browser compatibility, what do we do with this?

Reply:
Good call, since it is a display system, I can imagine you want to use it on 'simpeler' machines like raspberry PI and the like. Also, schools are sometimes stuck on older versions.

Reply:
Very good catch. I think we'd better avoid it for now: http://caniuse.com/#search=arrow